pkgs/development/libraries/libraylib/default.nix

```diff
@@ -0,0 +1,36 @@
+with import <nixpkgs> { };
+let
+  version = "2.6.0";
+  name = "raylib-${version}";
+  hash = "494e95eaf8daf3f086116dfd6a5fd2e9a2166fc744eabf4f3067bf887d4fb5ef";
+in
+stdenv.mkDerivation {
+  inherit name;
+
+  meta = {
+    description = "A simple and easy-to-use library to enjoy videogames programming";
+    repo = "raylib";
+    owner = "raysan5";
+    license = stdenv.lib.licenses.zlib;
+    maintainers = [ stdenv.lib.mantainers.svantepolk ];
+    rev = version;
+    sha256 = hash;
```

Comment:
I'm not familiar with some of these [meta-attributes](https://nixos.org/nixpkgs/manual/#chap-meta). Am I missing something?

Reply:
Mistakenly merged from a prior git source configuration I was trying. I've addressed this in a fixup.